src/App.js

```diff
@@ -1,29 +1,38 @@
 import React, { Component } from 'react'
 import { Provider } from 'react-redux'
-import { BrowserRouter as Router, Route, Link } from 'react-router-dom'
+import { BrowserRouter as Router, Route } from 'react-router-dom'
 
 import { login } from './services/firebase'
 import { createAppStore } from './store'
 import Navbar from './components/Navbar'
 import DiaryList from './routes/DiaryList'
 import Diary from './routes/Diary'
+import NewDiaryForm from './routes/NewDiaryForm'
+import { setItem, fetchDiaries } from './services/localStorage'
```

Comment:
I think if we can make this `./services/diaryApi` then in THAT file we use localStorage it would make swapping them out that much easier.

Reply:
makes sense